mev-programs/programs/tip-distribution/Cargo.toml

```diff
@@ -17,8 +17,8 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
-anchor-lang = "0.27.0"
+anchor-lang = "0.28.0"
 default-env = "0.1.1"
-jito-programs-vote-state = "0.1.3"
-solana-program = "1.14"
+jito-programs-vote-state = { path = "../vote-state" }
```

Comment:
fine to do this here but for future reference:
in order to publish this package to crates.io we'll need to first publish a new version of jito-programs-vote-state then point this at that version, since crates doesn't like it when your package references relative paths